sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -264,7 +264,7 @@ for (int i = 0; i < receiptPageResults.size(); i++) {
     System.out.printf("----------- Recognized Receipt page %d -----------%n", i);
     FormField<?> merchantNameField = recognizedFields.get("MerchantName");
     if (merchantNameField != null) {
-        if (FieldValueType.STRING.equals(merchantNameField.getValueType())) {
```

Comment:
is that because FieldValueType.STRING is static value, so you intend to comparing values by mem address value?

Reply:
Both work actually. But equals will just go and do `==` anyway so updates. Also, == will be null safe.